openml/flows/functions.py

```diff
@@ -167,8 +262,8 @@ def __list_flows(api_call):
     return flows
 
 
-def _check_flow_for_server_id(flow):
-    """Check if the given flow and it's components have a flow_id."""
+def _check_flow_for_server_id(flow: OpenMLFlow) -> None:
+    """ Raises a ValueError of the flow or any of its subflows has no flow id. """
```

Comment:
for the flow, if any typos

Reply:
Changed it to `Raises a ValueError if the flow or any of its subflows has no flow id.`, good catch.